### ISSUE TITLE
Trailing whitespace while marking column name

### ISSUE THREAD

**Issue by nook24** [crate/crate-admin#559]
This is not really an issue and I guess im the only one who is affected by this...
How ever, if you mark an column name, there is a trailing whitespace which needs to be removed manually.
![bildschirmfoto 2018-04-13 um 10 40 21](https://user-images.githubusercontent.com/9019992/38725402-84454a98-3f07-11e8-873d-0ce58b845164.png)

If you need to copy and paste a few columns that can be a bit annoying.
````php
$data = [
    'ack_author ' => $this->foo,
    'ack_data ' => $this->bar
];
````

I am on CrateDB 2.3.2

**Comment by m-kharbat** [comment]
Thank you for reporting, we will look into this and get back to you

**Comment by nook24** [comment]
Ok, due to the trailing space, I smashed my table:
![bildschirmfoto 2018-04-13 um 16 38 51](https://user-images.githubusercontent.com/9019992/38741118-782b609e-3f39-11e8-8022-5d304e527e82.png)
![bildschirmfoto 2018-04-13 um 16 41 25](https://user-images.githubusercontent.com/9019992/38741142-88db7e88-3f39-11e8-9f36-48395ca0ee97.png)

I created a json file with the space in the column name, and it gots auto created by CrateDB
![bildschirmfoto 2018-04-13 um 16 39 20](https://user-images.githubusercontent.com/9019992/38741153-8cddf5e2-3f39-11e8-87fd-feb975331d3d.png)

May be this is a good time to create the `ALTER TABLE statusengine_service_acknowledgements DROP COLUMN "service_description ";` feature request 😀

**Comment by m-kharbat** [comment]
That's another issue 😄, I made a ticket to discuss having more strict column name validation.

**Comment by nook24** [comment]
> I made a ticket to discuss having more strict column name validation

Found this:
`insert into foobar("😀")values('😀');`
Result:
![bildschirmfoto 2018-04-17 um 16 36 10](https://user-images.githubusercontent.com/9019992/38876783-7a712164-425d-11e8-9960-1b61bbbef01b.png)

I need to say, I'm not sure if i like this behavior or not 😀

**Comment by mfussenegger** [comment]
> I need to say, I'm not sure if i like this behavior or not grinning

Dynamic column creation on top-level of a table can be disabled by using `create table ... with (column_policy = 'strict') ` or for an existing table: `alter table table_name set (column_policy = 'strict');`

**Comment by m-kharbat** [comment]
this issue has been fixed and will be in the next 2.3 release.
